deploy/update-ltv.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
```

Comment:
Lets not add the kubernetes deployments here. It's better to have a periodic tasks helm chart here: https://github.com/stakewise/helm-charts/

Reply:
FYI: there is manifest.yml file in multichain repo, contains kubernetes stuff.

Reply:
We should remove that from there and move to helm charts